src/core/Crust/Infobox/Block/HTML/index.tsx

```diff
@@ -0,0 +1,237 @@
+    setEditingText(html ?? "");
+  }, [isEditable, html]);
+
+  const finishEditing = useCallback(() => {
+    if (!isEditing) return;
+    if (onChange && isDirty.current) {
+      onChange("default", "html", editingText ?? "", "string");
+    }
+    isDirty.current = false;
+    setEditingText(undefined);
+  }, [editingText, onChange, isEditing]);
+
+  const handleChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      setEditingText(e.currentTarget.value);
+      isDirty.current = true;
+    },
+    [],
+  );
+
+  useEffect(() => {
+    if (isEditing) {
+      ref.current?.focus();
+    }
+  }, [isEditing]);
+
+  const isSelectedPrev = useRef(false);
+  useEffect(() => {
+    if (isEditing && !isSelected && isSelectedPrev.current) {
+      finishEditing();
+    }
+  }, [finishEditing, isSelected, isEditing]);
+  useEffect(() => {
+    isSelectedPrev.current = !!isSelected;
+  }, [isSelected]);
+
+  const [isHovered, setHovered] = useState(false);
+  const handleMouseEnter = useCallback(() => setHovered(true), []);
+  const handleMouseLeave = useCallback(() => setHovered(false), []);
+  const handleClick = useCallback(
+    (e: React.MouseEvent<HTMLDivElement>) => {
+      e.stopPropagation();
+      if (isEditing) return;
+      onClick?.();
+    },
+    [isEditing, onClick],
+  );
+
+  useLayoutEffect(() => initializeIframe(), [initializeIframe]);
+
+  useEffect(() => {
+    const handleMessage = (e: any) => {
+      const data: MessageEventData = e.data;
+      if (data.type === "mount") {
+        setHeight(data.height);
+      }
+    };
+    window.addEventListener("message", handleMessage);
+
+    return () => {
+      window.removeEventListener("message", handleMessage);
+    };
+  }, []);
+
+  return (
+    <Wrapper
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+      onClick={handleClick}
+      isSelected={isSelected}
+      isHovered={isHovered}
+      isEditable={isEditable}
+      isTemplate={isTemplate}>
+      {isTemplate && isEditable && !isEditing ? (
+        <Template onDoubleClick={startEditing}>
+          {/* FIXME(@keiya01): Use HTML icon */}
+          <StyledIcon icon="text" isSelected={isSelected} isHovered={isHovered} size={24} />
+          <Text isSelected={isSelected} isHovered={isHovered}>
+            {t("Double click here to write.")}
+          </Text>
+        </Template>
+      ) : (
+        <>
+          {title && <Title>{title}</Title>}
+          {isEditing ? (
+            <InputField
+              ref={ref}
+              value={editingText ?? ""}
+              onChange={handleChange}
+              onBlur={finishEditing}
+              rows={10}
+              minHeight={height}
+            />
+          ) : (
+            <IFrame
+              key={html}
+              ref={setFrameRef}
+              onDoubleClick={startEditing}
+              srcDoc={html}
+              $height={height}
+              allowFullScreen
```

Comment:
```suggestion
              allowFullScreen
              sandbox="allow-same-origin allow-popups allow-forms"
```

refer to https://github.com/CesiumGS/cesium/blob/1.102/packages/widgets/Source/InfoBox/InfoBox.js#L70

Reply:
But we need also `allow-scripts`. When I add `allow-scripts` then chrome suggest sandbox is no longer needed.
So I didn't add sandbox.

Reply:
Hmmm, that would make XSS easy. On the other hand, I need to be able to play videos with relative paths. I don't know what to do...

Reply:
I fixed
https://github.com/reearth/reearth-web/pull/454/commits/af0772d77dbd1feeef87f35ad2e994ac588bc6b3